Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -82,7 +84,9 @@ public class UsersModule : FabricModule<User>
             await SetDefaultRequest(userPermissionRequest);
             await CheckAccess(_clientService, userPermissionRequest.Grain, userPermissionRequest.SecurableItem,
                 AuthorizationReadClaim);
-            var groups = GetGroupsForAuthenticatedUser();
+
+            var subjectId = Context.CurrentUser.Claims.First(c => c.Type == Claims.Sub).Value;

```

Comment:
Should we push this to a function in FabricModule?

Reply:
@michaelvidal24 , are you referring to the entire GetCurrentUserPermissions() method?

Reply:
@bsmithvt, I was referring to just line 88:

```
var subjectId = Context.CurrentUser.Claims.First(c => c.Type == Claims.Sub).Value;
```

Seems like it is a candidate to be reused elswhere  in the authorization api so might be helpful to have in FabricModule.